packages/localize/src/internal/localized-decorator.ts

```diff
@@ -36,28 +43,18 @@ import type {
  *     }
  *   }
  */
-const _localized =
-  () => (classOrDescriptor: Constructor<ReactiveElement> | ClassDescriptor) =>
-    typeof classOrDescriptor === 'function'
-      ? legacyLocalized(classOrDescriptor as unknown as typeof ReactiveElement)
-      : standardLocalized(classOrDescriptor);
-
-export const localized: typeof _localized & {
-  _LIT_LOCALIZE_DECORATOR_?: never;
-} = _localized;
-
-const standardLocalized = ({kind, elements}: ClassDescriptor) => {
-  return {
-    kind,
-    elements,
-    finisher(clazz: typeof ReactiveElement) {
-      clazz.addInitializer(updateWhenLocaleChanges);
-    },
+export const localized: Localized =
+  (): LocalizedDecorator =>
+  (
+    clazz: typeof ReactiveElement,
```

Comment:
Should we use the `Interface` trick to strip off non-public fields?

Reply:
done